### PR TITLE
added muted/audible toggler + fix styles/props admincheckbox

### PR DESCRIPTION
**Pull request by alexey-petrov**
Closes:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1214

* Fixed muted/audible text strings display
* Added `muted/audible` toggler to all iframe spaces
* fixed `AdminCheckbox` `disabled` prop
* Added option to have both `labelBefore` and `labelAfter` for `AdminCheckbox` input

Adding missing `muted/audible` toggle to basic iframe spaces (including new auditorium)
Adding ability to have both `before` and `after` labels on `AdminCheckbox` toggler
image

### CODE REVIEW

File: src/components/molecules/AdminCheckbox/AdminCheckbox.scss [hunk 1]
```diff
@@ -47,6 +47,10 @@ $checkbox-shadow-preset: inset 0 0 0 1px;
   &__label {
     display: flex;
     align-items: center;
+
+    &--before {
+      margin-right: $spacing--md;
+    }
```
Comment (goran-peoski-work) [inline src/components/molecules/AdminCheckbox/AdminCheckbox.scss:53]: Why this change? Can haz screenshots in descripshon? 🐱

File: src/components/molecules/AdminCheckbox/AdminCheckbox.tsx [hunk 2]
```diff
@@ -14,7 +14,8 @@ export interface AdminCheckboxProps
   className?: string;
   errors?: FieldErrors<FieldValues>;
   label?: ReactNode | string;
-  labelPosition?: "before" | "after";
+  labelAfter?: string;
+  labelBefore?: string;
```
Comment (goran-peoski-work) [inline src/components/molecules/AdminCheckbox/AdminCheckbox.tsx:18]: What happens when both are set to true?
This is not an orthogonal thing where 4 combinations are equally valid, but an enumeration of two values. 

```suggestion  
  labelPosition?: "before" | "after";
```

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
I don't see justified reason to "fix" the `AdminCheckbox` component

File: src/components/molecules/AdminCheckbox/AdminCheckbox.tsx [hunk 3]
```diff
@@ -10,25 +10,27 @@ import { CheckboxProps } from "components/atoms/Checkbox";
 import "./AdminCheckbox.scss";
 
 export interface AdminCheckboxProps
-  extends Omit<CheckboxProps, "label" | "toggler"> {
+  extends Omit<CheckboxProps, "label" | "toggler" | "flip-switch"> {
   className?: string;
   errors?: FieldErrors<FieldValues>;
   label?: ReactNode | string;
   labelAfter?: string;
```
Comment (goran-peoski-work) [inline src/components/molecules/AdminCheckbox/AdminCheckbox.tsx:17]:
```suggestion
```

File: src/components/molecules/AdminCheckbox/AdminCheckbox.tsx [hunk 4]
```diff
@@ -63,15 +65,19 @@ export const AdminCheckbox: React.FC<AdminCheckboxProps> = ({
...
+      {isInputWithLabel ? (
         <label className="AdminCheckbox__label">
-          {labelBefore && (
-            <span className="AdminCheckbox__label--before">{labelBefore}</span>
+          {isToggleFlip && (
+            <span className="AdminCheckbox__label--before">{textOff}</span>
           )}
           {input}
           {label && label}
+          {isToggleFlip && textOn}
```
Comment (goran-peoski-work) [inline src/components/molecules/AdminCheckbox/AdminCheckbox.tsx]: Not sure if  messing  with the label is the right way to go. Probably leave it on its own so `textOn` and `textOff` should be visible regardless if there is a label or not. 

I'm thinking of a scenario that might have a label on top from the toggle instead from a section, but I will have to do a little local check to know for sure.

File: src/components/molecules/SpaceEditForm/SpaceEditForm.scss [hunk 5]
```diff
@@ -24,6 +24,16 @@
     padding: $spacing--xl;
   }
 
+  &__input {
+    &-reactionMute {
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditForm/SpaceEditForm.scss:28]: No camel case, kebob here. 

We reserve the camels only for the B of B.E.M. Please also do not start any selector with `&-` (unless rare corner case demands) as people in a rush might identify it as the more common `&__` and it does add restriction to the class naming and discovery. 

There is not much to gain of keeping style DRY-est possible. Having only few elements, not really much of a help putting the `input` in there.

```suggestion
    &__input-reaction-mute {
```

File: src/components/molecules/SpaceEditForm/SpaceEditForm.scss [hunk 6]
```diff
@@ -24,6 +24,16 @@
     padding: $spacing--xl;
   }
 
+  &__input {
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditForm/SpaceEditForm.scss:27]: Are the selectors under this `&__input` even used anywhere?

File: src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss [hunk 7]
```diff
@@ -21,6 +21,16 @@ $textarea-height: 120px;
...
+  &__input {
+    &-reactionMute {
+      display: flex;
+    }
+
+    &-subtext {
+      margin-right: $spacing--md;
+    }
+  }
+
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss:33]: Are these even used anywhere? If not please remove, and if so, please do refactor as per above comment.

File: src/components/molecules/AdminCheckbox/AdminCheckbox.tsx (hunk 4 above)
Comment (goran-peoski-work) [reply src/components/molecules/AdminCheckbox/AdminCheckbox.tsx]: I think there is a way to untangle this, have something done, just need to check if it works: https://github.com/sparkletown/sparkle/commit/aef7642add90dbcc95f735091b3e13bb5c68afc8#diff-1bb7cbf10026a360bed006fc8713a20180ac64d4844f4946e0444415dacf9d1b

**Review by goran-peoski-work — CHANGES_REQUESTED** [review]
`AdminCheckbox` is still unclear about what is a label and what is a helper text. On/Off should be considered visual aid, part of the toggle display, not the label and, who knows, maybe in the future instead of text it can be emoji or svg.

File: src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss (hunk 7 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss]: removed ef337b8

File: src/components/molecules/SpaceEditForm/SpaceEditForm.scss (hunk 6 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditForm/SpaceEditForm.scss]: removed ef337b8

File: src/components/molecules/SpaceEditForm/SpaceEditForm.scss (hunk 5 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditForm/SpaceEditForm.scss]: removed ef337b8

File: src/components/molecules/AdminCheckbox/AdminCheckbox.tsx (hunk 3 above)
Comment (alexey-petrov) [reply src/components/molecules/AdminCheckbox/AdminCheckbox.tsx]: removed ef337b8

File: src/components/molecules/SpaceEditForm/SpaceEditForm.tsx [hunk 8]
```diff
@@ -437,12 +446,31 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
...
+                  <>
+                    <AdminCheckbox
+                      name="venue.showReactions"
+                      label="Show reactions"
+                      variant="toggler"
+                      register={register}
+                    />
+                    {!isReactionsMutedDisabled && (
+                      <AdminSection>
+                        <div className="mod--flex-row">
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditForm/SpaceEditForm.tsx:458]:
```suggestion
                        <div className="SpaceEditForm__reactions-muted">
```

File: src/components/molecules/SpaceEditForm/SpaceEditForm.scss [hunk 9]
```diff
@@ -28,4 +28,15 @@
     // NOTE: Magic number determined empirically to fill the footer space in just this one case
     width: 2 * $admin-ng-sidebar-footer-buttons-width--magic-number;
   }
+
+  .AdminSection {
+    .mod--flex-row {
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditForm/SpaceEditForm.scss:33]:
```suggestion
    &__reactions-muted {
```

File: src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss [hunk 10]
```diff
@@ -31,6 +31,14 @@ $textarea-height: 120px;
 
   .AdminSection {
     padding: $spacing--sm $spacing--xl;
+    .mod--flex-row {
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss:34]:
```suggestion
    &__reactions-muted {
```

File: src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx [hunk 11]
```diff
@@ -345,15 +347,23 @@ export const SpaceEditFormNG: React.FC<SpaceEditFormNGProps> = ({
...
-              name="isReactionsMuted"
-              forwardedRef={register}
-              disabled={!values.showReactions}
-              containerClassName="SpaceEditFormNG__toggler"
-              label="Audible"
-            />
-          </AdminSection>
+          {values.showReactions && (
+            <AdminSection>
+              <div className="mod--flex-row">
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx:352]:
```suggestion
              <div className="SpaceEditFormNG__reactions-muted">
```

**Review by goran-peoski-work — APPROVED** [review]
Approved.

Please change the class name before merge, `mod--` classes are good for rapid prototyping and playing with while developing, less so in production (same issues with `global.scss` and the likes).

File: src/components/molecules/SpaceEditForm/SpaceEditForm.tsx (hunk 8 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditForm/SpaceEditForm.tsx]: removed & replaced with new `AdminCheckbox` c891a3f

File: src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss (hunk 10 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss]: removed & replaced with new `AdminCheckbox` c891a3f

File: src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx (hunk 11 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.tsx]: removed & replaced with new `AdminCheckbox` c891a3f

File: src/components/molecules/SpaceEditForm/SpaceEditForm.scss (hunk 9 above)
Comment (alexey-petrov) [reply src/components/molecules/SpaceEditForm/SpaceEditForm.scss]: removed & replaced with new `AdminCheckbox` c891a3f

**Review by ermiaSparkle — COMMENTED** [review]
@alexey-petrov Please also fix the client-side of audible.

**Comment by alexey-petrov** [comment]
@ermiaSparkle done. please re-check the `ReactionsBar` mute icon corresponding to reactions mute flag in admin panel

File: src/components/molecules/AdminCheckbox/AdminCheckbox.tsx [hunk 12]
```diff
@@ -80,6 +80,8 @@ export const AdminCheckbox: React.FC<AdminCheckboxProps> = ({
 
   return (
     <p className={parentClasses}>
+      {/* @debt: Need to fix SCSS to make {input} tick/toggle work without provided label*/}
+      {/* Currently if no label is provided checkbox/toggle don't change their state */}
```
Comment (goran-peoski-work) [inline src/components/molecules/AdminCheckbox/AdminCheckbox.tsx:84]: This has been fixed in another PR, safe to remove here.

```suggestion
```

**Review by goran-peoski-work — APPROVED** [review]
LGTM.

Can probably apply the suggestion about removing the comment before merge. Now `AdminCheckbox` will always have at least one utility label on its own w/o needing outside label to work.

**Comment by alexey-petrov** [comment]
@ermiaSparkle re-tested this one locally and at https://co-reality-staging--preview-pr-2540-y8n66qv9.web.app/ and confirmed that new auditorium and other iframe spaces that have some embedded content display reactions' audible/muted flag properly, so might need another qa round from you

**Review by ermiaSparkle — APPROVED** [review]
Will be QA in staging.
